README.md

```diff
@@ -139,6 +139,7 @@ The following table shows the supported combinations:
 
 | LiT               | LND          | Loop         | Faraday       | Pool         | Taproot Assets |
 |-------------------|--------------|--------------|---------------|--------------|----------------|
+| **v0.10.2-alpha** | v0.16.4-beta | v0.24.1-beta | v0.2.11-alpha | v0.6.4-beta  | v0.2.2-alpha   |
```

Comment:
I think we should include this in the commit that's bumping the LiT version instead, and then have that commit after the different dependencies bumps :).